packages/output/src/utils/styles.js

```diff
@@ -152,6 +152,14 @@ function CustomStyles() {
                 margin-bottom: 16px;
                 text-align: center;
               }
+
+              amp-story-audio-sticker {
+                height: 100%;
+              }
+
+              amp-story-grid-layer .i-amphtml-amp-story-audio-sticker-tap-hint {
```

Comment:
This causes the AMP validation error because we are using `.i-amphtml-...` selectors in CSS.

We need to find another selector to fix the font family.

Reply:
@swissspidy I am still getting this validation error in debug panel when previewing story:

<img width="1348" alt="image" src="https://github.com/GoogleForCreators/web-stories-wp/assets/75766877/6f1156b0-ce63-4673-8c19-1bf2f7da8e26">

```
The parent tag of tag 'amp-story-audio-sticker' is 'div', but it can only be 'amp-story-grid-layer'.
```

I think, https://github.com/ampproject/amp-wp/pull/7741 is yet to release with milestone `2.5.3`. Current release is `2.5.2` which was released back in January, 2024.

> This causes the AMP validation error because we are using .i-amphtml-... selectors in CSS.
> We need to find another selector to fix the font family.

I am finding another way to fix this one.

Reply:
@AnuragVasanwala The debug panel has nothing to do with the AMP plugin. It just means that the public AMP validator used by the runtime is not yet updated. That should just be a matter of time I think.

Updating the AMP plugin is only relevant for our sanitization logic and PHP unit tests.

The AMP plugin release also does not matter as we directly use the develop branch: 

https://github.com/GoogleForCreators/web-stories-wp/blob/e146a96203412f16f98863b7a1de86c095b94522/composer.json#L15